tests/tensorflow2/test_get_hook.py

```diff
@@ -0,0 +1,53 @@
+# Standard Library
+from statistics import mean
+
+# Third Party
+from tensorflow.python.util.smdebug import get_smdebug_hook
+from tests.utils import SagemakerSimulator, Timer
+
+RANGE = 1000000  # 1 Million
+REPEAT = 5
+
+
+def test_get_smdebug_hook_no_setup():
+    times_taken = []
+    for _ in range(REPEAT):
+        timer_context = Timer()
+        with timer_context as t:
+            for _ in range(RANGE):
+                hook = get_smdebug_hook("keras")
+        times_taken.append(t.time_taken)
+
+    mean_time_taken = mean(times_taken)
+    print(mean_time_taken)
+    assert mean_time_taken < 16  # current mean = ~14.5 seconds
```

Comment:
How did we decide on 16?

Reply:
What threshold should we go for here? 15 maybe?